tests/e2e/local/vagrant/Vagrantfile

```diff
@@ -82,7 +82,7 @@ Vagrant.configure("2") do |config|
      sudo tar -xf go.linux-amd64.tar.gz
      sudo mv go /usr/local
      sudo rm go.linux-amd64.tar.gz
-     kubectl apply -f /istio.io/istio/tests/util/localregistry/localregistry.yaml
+     kubectl apply -f /istio.io/istio/tests/util/registry/k8s_registry.yaml
```

Comment:
IIRC, we're going to get rid of Vagrant soon anyway.  I think we should leave this yaml file as-is, but make a copy of it for our new registry code (so we can change it to a template).

Reply:
We want the test framework to push its own cluster. We can (and do) use namespaces as a unit of isolation. The pilot e2e tests today can use UUIDs for namespaces to allow us to run multiple istios on the same cluster.  The local registry should support this feature.